modules/controlfreec/CHANGELOG.md

```diff
@@ -38,4 +38,22 @@ This release was authored by Jasper.
 
 Updated controlfreec.smk to allow for running on Numbers cluster.
 
-Fixed bugs related to genome builds and preparation of controlfreec-specific reference files. Now will generate chr.len file for hg19, hg38 (w/ chr prefix), grch37, grch38 (no chr).
\ No newline at end of file
+Fixed bugs related to genome builds and preparation of controlfreec-specific reference files. Now will generate chr.len file for hg19, hg38 (w/ chr prefix), grch37, grch38 (no chr).
+
+## [1.2] - 2020-09-10
+
+This release was authored by Jasper.
+
+Fixed a config bug (snakemake replacing 3&4 with 3numdegree4 in config_WGS.txt) - added \ on top of &. Note: tested it, the output was the same (since 3&4 is default)
+Changed output dir to be controlfreec-1.1/ instead of controlfreec-1.0/
+Added a new feature in snakemake to allow for conversion of CNV calls to bed.
+
+## [1.2] - 2020-11-05
+
+This release was authored by Jasper.
+
+Improved control-freec calls for FFPE genomes. Everything is now default in paired mode with BAF turned on to measure LOH/BAF. All unmatched samples will be ran with a default normal. This will now generate temporary pileups in a scratch space (time consuming). The parameters have been set to minimize noise while also maximizing the number of prominent signals across the genome. To accomplish this, I set the default ploidy to 2 to prevent control-freec from overfitting the model for unpaired samples when ploidy != 2. Importantly, I set the parameter forceGCcontentNormalization=1, which forces GC normalzation for both the sample and control read counts (RC), and then calculates the ratio of sample RC/control RC. Then I changed the intercept of the polynomial from 1 to 0 to make the valuation of sample relative to the control instead of to its own GC content. This effectively smoothed out the profiles without added segments of overamplified "noise". I also allowed control-freec to assess the relative normal contamination and allowed control-freec to set its own window size given the coverage across the genome. These parameters effectively smoothed out unnecessary noise even further.
```

Comment:
` All unmatched samples will be ran with a default normal.`
How about matched samples? Are they running paired with patient's normal, or the same default normal as unpaired samples?

Reply:
Patient's normals. I set the config to be:

`        pairing_config:
            genome:
                run_paired_tumours: True
                run_unpaired_tumours_with: "unmatched_normal"
                run_paired_tumours_as_unpaired: False`

Reply:
Ideally, since control freec is able to run without a normal, the module would be user-configurable to run without one.

Reply:
Agree. Either we should add a comment to the default config file to explain what options should be used with an unmatched normal or the module should be able to set the appropriate combination of parameters depending on whether its being run in inpaired, unmatched or matched mode.